invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -80,7 +80,13 @@ class SearchResultItemComponent extends Component {
           collapsing
           className="word-break-all"
         >
-          {(recordOwner && recordOwner.user) || "System"}
+          {recordOwner ? (
+            <a rel="noreferrer" href={`/administration/users?q=id:${recordOwner.user}`}>
+              {recordOwner.user}
+            </a>
+          ) : (
+            i18next.t("System")
```

Comment:
What is the use case for that? I.e. what kind of record do not have owners?

Reply:
I'm just persisting the existing logic 😇 no idea, but I guess it's better than having it be empty 🤷

Reply:
LGTM, but any reason for noreferrer?

Reply:
I think this might be messing up the analytics tool if anyone is analyzying the user behaviour within the application. To be checked, but if it is the case, I would suggest not to follow the habit further.

Reply:
I'm not aware that we're tracking user behaviour on the administration pages (or really anywhere) but noted

Reply:
> It's not necessary no, but we have it on L67 in this file and it is a general habit of mine

@carlinmack @kpsherva @zzacharo 
The `noreferrer` is necessary when opening in a new window/tab, when having `target="_blank"`. And actually, we are missing `noopener`: 
`rel="noopener noreferrer"`

Can you please add it above, and remove the `noreferrer` from here if you don't want to open it in a new window/tab?

More info: https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

Reply:
thanks for the link and explanation!

Reply:
removed the `noreferrer`, will handle the other one in https://github.com/inveniosoftware/invenio-app-rdm/pull/2894 to not create conflicts